test/worlds/particle_emitter2.sdf

```diff
@@ -0,0 +1,150 @@
+<?xml version="1.0" ?>
+<sdf version="1.6">
+  <world name="particle_emitters">
+
+    <physics name="1ms" type="ode">
+      <max_step_size>0.001</max_step_size>
+      <real_time_factor>1.0</real_time_factor>
+    </physics>
+
+    <plugin
+      filename="ignition-gazebo-particle-emitter2-system"
+      name="ignition::gazebo::systems::ParticleEmitter2">
+    </plugin>
+
+    <gui fullscreen="0">
+
+      <!-- 3D scene -->
+      <plugin filename="GzScene3D" name="3D View">
+        <ignition-gui>
+          <title>3D View</title>
+          <property type="bool" key="showTitleBar">false</property>
+          <property type="bool" key="resizable">false</property>
+          <property type="double" key="z">0</property>
+          <anchor line="right" target="window" target_line="right"/>
+          <anchor line="left" target="window" target_line="left"/>
+          <anchor line="top" target="window" target_line="top"/>
+          <anchor line="bottom" target="window" target_line="bottom"/>
+        </ignition-gui>
+
+        <engine>ogre</engine>
```

Comment:
I think we don't need the gui for the test file but in case we want to see the particles in the gui, we should use `ogre2` here.

Reply:
Updated in 8a6478207e6ebc0522c0fd8e4531a8ce1c0ff7c5